packages/db/.kanelrc.js

```diff
@@ -16,6 +22,38 @@ module.exports = {
 
 	preDeleteOutputFolder: false,
 	enumStyle: "enum",
+	generateIdentifierType: (column, details, config) => {
+		const name = escapeIdentifier(toPascalCase(details.name) + toPascalCase(column.name));
+		const innerType = resolveType(column, details, {
+			...config,
+			// Explicitly disable identifier resolution so we get the actual inner type here
+			generateIdentifierType: undefined,
+		});
+		const imports = [];
+
+		let type = innerType;
+		if (typeof innerType === "object") {
+			// Handle non-primitives
+			type = innerType.name;
+			imports.push(...innerType.typeImports);
+		}
+
+		let typeDefinition = [`${type} & { __brand: '${escapeString(name)}' }`];
+
+		/* this is the custom part */
+		if (column.type.kind === "enum") {
+			typeDefinition = [`${type}`];
+		}
```

Comment:
Without this change, kanel generates branded types for the three id columns in `membership_capabilities`. That's not really correct IMO since the point of making those columns use enums is that they are _not_ unique to the table, and are in fact equivalent to any other column with that same enum type.